_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -16,6 +16,7 @@ class HealthCareApp extends React.Component {
     this.publishStateChange = this.publishStateChange.bind(this);
     this.handleContinue = this.handleContinue.bind(this);
     this.getNextUrl = this.getNextUrl.bind(this);
+    this.getExternalData.bind(this);
```

Comment:
I think this should be like the others above: `this.getExternalData = this.getExternalData.bind(this);`